### PR TITLE
Changes in Read.me

### LINKED ISSUE #8: docs: update README
This issue focuses on creating a proper README.md file for this project. At this point I think it is a better option to include the contents of `contributing.md` into `readme.md` so that developers can access the contents easily.

### PR DESCRIPTION
**Pull request by RicardoHaranoP**
contributing.md into read.me

fixes #8

### CODE REVIEW

**Comment by RamGoel** [comment]
@RicardoHaranoP Thanks for your contribution. Can you please change the project name from `cdnExtension` to `easyCDN` and also mention other contributors in the contributor section

File: README.md [hunk 1]
```diff
@@ -7,6 +7,70 @@ Features-
...
+Now submit the pull request.
+
+
+
+
+Happy Contributing :)
+
+
 ## contributors
 [Ram Goel](https://github.com/RamGoel)
```
Comment (RamGoel) [inline README.md]: Please add contributor list here

**Comment by RicardoHaranoP** [comment]
hello @RamGoel, like that?
`
# easyCDN

No more Searching for CDN Links again and again on the Web. Our Tool cdnExtension solves this problem by keeping all the popular cdn links right on the top of your browser.

## contributors
- [Ram Goel](https://github.com/RamGoel)
- [Ankit Singh](https://github.com/Mahisingh09)
- [janhvi pandey](https://github.com/jpgJanhvi)
- [Rohit Patil](https://github.com/rohit-raje-786)
`

**Comment by RamGoel** [comment]
Fixed in #12

**Pull request closed without merging**